feature-libs/organization/administration/styles/_card.scss

```diff
@@ -93,17 +96,35 @@
     margin: 40px;
     background-color: var(--cx-color-inverse);
     border: solid 1px var(--cx-color-light);
+    @include cx-highContrastTheme-light {
+      border: 1px solid rgba(0, 0, 0, 0.125);
```

Comment:
`query:` maybe we should consider moving `rgba(0, 0, 0, 0.125)` to a variable?

Reply:
Currently, we don't have a global common variable for the border color, unless we decide to introduce a local one.